bobtemplates/plone_addon/README.rst.bob

```diff
@@ -2,9 +2,10 @@
    If you want to learn more about writing documentation, please check out: http://docs.plone.org/about/documentation_styleguide.html
    This text does not appear on pypi or github. It is a comment.
 
-==============================================================================
+{{% for i in package.dottedname %}}={{% endfor %}}
+
```

Comment:
@laulaz if you have tested it, then discard my comment, I thought that it would have an extra empty line in between

thanks for double checking!